cfgov/unprocessed/css/organisms/snippet-list.less

```diff
@@ -0,0 +1,44 @@
+/* =========================================================================
+   cfgov-refresh
+   o-snippet-list
+   ========================================================================= */
+
+.o-snippet-list_list {
+    display: table;
+    width: 100%;
+    margin-top: unit( 30px / @base-font-size-px, em );
+    border-collapse: collapse; // must be set to allow borders on table-rows
+
+    > ul {
```

Comment:
Should probably change this and the other element attributes below to the corresponding  `data-attr` to be consistent with the suggested changes to the cf-tables enhancement.